tools-and-communication/creating-videos.instructor.md

```diff
@@ -0,0 +1,76 @@
+# Video Guidelines
+
+This is a **brief** guide for instructors on how to create video lessons for students.
+
+## Video Tooling
+
+At Ada we have a few tools available for making video lessons:
+
+* **[Panopto](https://adaacademy.hosted.panopto.com/)** - Panopto is a company that provides lecture recording, screencasting and video streaming to Ada.  
+  * Most importantly all the storage space on the Panopto servers is free to Ada as a donation.  Panopto provides embed codes for all videos via an iframe tag.
+  * It also provides a [screencasting tool](https://adaacademy.hosted.panopto.com/Panopto/Cache/10.3.1.00010/Software/Panopto%20Recorder.pkg?arch=None&useCustomBinary=True) which can record the screen and webcam and automatically upload to panopto.  The tool is very _simple and basic_, but it does work and is easy to use.
+  * Panopto also allows us to string videos together into playlists.  This lets us make short vidoes covering specific topics in longer lessons.
+* **[Zoom](https://zoom.us)** - Zoom is our video conferencing software and it provides a way to record meetings which works pretty well.  
+  * You can record any meeting and download it to your computer and then upload the results to Panopto.  This is an outstanding way to do a _dual instructor lesson_.
+
+
+## In General
+
+- **Don't just read the lesson aloud**
+  - Remember that the video is a lesson, and does not have to involve reading the Learn curriculum line-by-line.  You can provide alternative examples of the same points.
+  - The key is you want to teach the same concepts as the Learn lesson, and not have a key concept taught in the video which is missing from the text.  Supplementary "neato" tricks are fine.
+- **Demonstrate Code!**
+  - Some of the text lessons are by necessity text-dense.  The nice thing about the video is you can explain the concept verbally while illustrating it with a code sample.
+- **Draw!**
+  - It can be really handy to use Zoom Annotations, or a [drawing app](https://app.diagrams.net/) to sketch out an illustration while you teach the concept aloud.  
+    - Feel free to add the finished product to the Learn lesson as well!  🐥🐥 for 🪨
+- **Come Back to the learning goals**
+  - Talk about the learning objectives in the beginning.
+  - Come back to them at the end and connect them to the topics discussed (great review)
+- **Keep it short!**
+  - The ideal video should be ~10 minutes.  Try never to go more than ~20 as that becomes a real slog for students.  It's also harder for them to focus in on a difficult concept if they have to hunt through the video to find things.
```

Comment:
I'm curious if there's a reason why 10 is the ideal. 
Namely, I'm curious whether break lessons down even further is nice, if the content can be broken down nicely, or if we think ~5 minutes actually feels too short. This actually came up as a question when Becca and I were recording yesterday.

Reply:
It's a rough and totally arbitrary number, and I probably should have phrased it as < ~10 minutes.

Less than 5 minutes and there's a bit of spin-up and spin-down on a topic that can make it a bit jarring and it adds to the recording time to make a bunch of small videos and link them up.  Plus the playlist gets long.

Reply:
That's valid! I'm down if you want to add that as well, but maybe you don't feel its necessary because nobody yet has had an issue of making videos too short...?